core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1131,7 +1131,7 @@ private void completeFileInternal(List<Long> blockIds, LockedInodePath inodePath
       FileAlreadyCompletedException, UnavailableException {
     InodeFile inode = inodePath.getInodeFile();
     inode.setBlockIds(blockIds);
-    inode.setLastModificationTimeMs(opTimeMs);
+    inode.setLastModificationTimeMs(opTimeMs, true);
```

Comment:
why did this have to change?

Reply:
Inode last modification time are initialized as `currentTimeMillis()` when they are created. To update it to a UFS timestamp that might be smaller requires setting the `override` argument to true here.

Reply:
I see. Can you add that type of comment for this line?